openvm/tests/apc_builder_outputs/single_or.txt

```diff
@@ -1,31 +0,0 @@
-// Bus 0 (EXECUTION_BRIDGE):
```

Comment:
Seems like this is not needed? I regenerated the snapshots and this one vanished.